src/engraving/libmscore/cmd.cpp

```diff
@@ -728,6 +731,23 @@ void Score::addInterval(int val, const std::vector<Note*>& nl)
 
         select(note, SelectType::SINGLE, 0);
     }
+    unsigned i = 0;
+    for (Note* on : nl) {
+        if (on->tieFor()) {
+            auto it = find(nl.begin(), nl.end(), on->tieFor()->endNote());
+            if (it != nl.end()) {
```

Comment:
Actually, the only thing you want to know here is whether the end note of the tie was included in the list of selected notes. We have a small convenience method for that in `containers.h`:
```suggestion
            if (mu::contains(nl, on->tieFor()->endNote()) {
```